object/tombstone.go

```diff
@@ -33,12 +33,18 @@ func (t *Tombstone) ToV2() *tombstone.Tombstone {
 	return (*tombstone.Tombstone)(t)
 }
 
-// ExpirationEpoch return number of tombstone expiration epoch.
+// ExpirationEpoch returns the last NeoFS epoch
+// number of the tombstone lifetime.
```

Comment:
Do you think `epoch number` is better than just `epoch`?

Reply:
took that naming from another comment, do not remember exactly where from